zipline/finance/blotter.py

```diff
@@ -33,30 +34,36 @@
 
 log = Logger('Blotter')
 
-class Blotter(object):
 
-    def __init__(self):
-        self.transact = transact_partial(VolumeShareSlippage(), PerShare())
```

Comment:
Why the removal here, besides clarity?
(In hindsight the `transact_partial` is a little too indirect.)